app/src/components/features/rules/RulePairs/Pairs/Rows/RowsMarkup/CustomScriptRow/index.js

```diff
@@ -47,8 +133,66 @@ const CustomScriptRow = ({ rowIndex, pairIndex, isLastIndex, deleteScript, scrip
   };
 
   const renderURLInput = () => {
+    const renderCodeEditorForScriptAttributes = () => {
```

Comment:
Can be very confusing.
`renderCodeEditorForScriptAttributes` inside  `renderURLInput`.
Can we have a common code editor component which can be wrapper for `CodeEditor` pass props accordingly for `script` and `url` type code editor.
This will also remove code duplication in both editor render functions